contentdm_exporter/contentdm_record_exporter.py

```
@@ -146,6 +147,41 @@ def get_list_of_collection_aliases():
+def create_current_mapping_table(collecton_aliases):
+    aliases_fields = {}
+
+    for alias in collecton_aliases:
+
+        query_url = "{main_url}dmGetCollectionFieldInfo/{alias}/{format}".format(
+            main_url=MAIN_URL, alias=alias, format="json"
+        )
+
+        # Query CONTENTdm and return records; if failure, log problem.
+        try:
+            req = requests.get(query_url)
```

Ruff (0.14.13)

Probable use of `requests` call without timeout

(S113)